api/_discord_bot.py

```diff
@@ -319,6 +326,56 @@ async def logged(ctx,
     await ctx.respond(f"Done")
 
 
+@bot.slash_command(name="blackhole", description="Blackholes", guild=discord.Object(id=1192396529705701477))
+async def blackhole(ctx, 
+                    timeline: Option(str, 'Future or Past', required=True, choices=(valid_timelines)), 
+                    days: Option(int, 'Days', required=False)):
+
+    await ctx.defer()
+
+    if days is None:
+        days = 5
+    datenow = datetime.datetime.now()
+    if timeline == "future":
+        datelimit = datenow + datetime.timedelta(days=days)
+        refer = executeQuerySelect("SELECT * FROM users WHERE has_cursus21 = True AND blackhole < %s AND blackhole > %s", (datelimit, datenow))
+    else:
+        datelimit = datenow - datetime.timedelta(days=days)
+        refer = executeQuerySelect("SELECT * FROM users WHERE has_cursus21 = True AND blackhole > %s AND blackhole < %s", (datelimit, datenow))
+
```

Comment:
use named parameter like https://github.com/Jerome-JJT/ftlwatch3/blob/e6293b2de4f9501bb9726bbb364551529f7844eb/api/update_users.py#L23

Reply:
genre ca ? :

`if timeline == "future":
        datelimit = datenow + datetime.timedelta(days=days)
        refer = executeQuerySelect("""SELECT * FROM users WHERE has_cursus21 = True AND blackhole < %(datelimit)s AND blackhole > %(datenow)s""", {
            "datelimit": datenow + datetime.timedelta(days=days),
            "datenow" : datetime.datetime.now()
        })`

Reply:
y